src/styles/blocks/main.css

```diff
@@ -0,0 +1,17 @@
+.body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  width: 100px;
+  height: 16px;
+  justify-content: space-around;
+}
+
+.stars__star {
+  background-image: url("../images/star-active.svg");
+  width: 16px;
+  background-repeat: no-repeat;
+  margin: 2px 0;
```

Comment:
add margin-right: 4px due to figma and remove margin-right from the last stars in a row